scss/sections/trade/_trade_respo.scss

```diff
@@ -49,43 +70,59 @@
 
 @media (min-width: 600px) {
     .trade {
-        .trade_header {
+        .navyblue_section {
             p {
```

Comment:
why u overwrite p/h1/h2 etc in each section they are global selectors!!! move this selectors to media queries to general directory for this selectors!!